### PR TITLE
CC-11339 : Support kerberos authentication for splunk sink connector

### PR DESCRIPTION
**Pull request by venkatteki**
Add kerberos authentication support for splunk sink connector 
Add 2 new configs:

1. kerberos.user.principal
2. kerberos.keytab.path

### CODE REVIEW

**Comment by venkatteki** [comment]
Future improvements for this change are being tracked as part of CC-13494

File: pom.xml [hunk 1]
```diff
@@ -152,6 +152,11 @@
             <version>3.7</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>io.confluent</groupId>
```
Comment (ragepati) [inline pom.xml]: I believe this is an internal repo. I don't think it will build outside Confluent.

Comment (venkatteki) [reply pom.xml]: Removed this dependency

Comment (venkatteki) [reply pom.xml]: > I believe this is an internal repo. I don't think it will build outside Confluent.

Removed the dependency. Thanks for pointing out

**Pull request closed without merging**